uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document.controller.spec.js

```diff
@@ -614,9 +614,8 @@ define([
       });
     });
 
-    describe('when user click on the document wrench icon', function () {
-      var onPopupFormSuccess;
-      var url;
+    describe('when user clicks on the document wrench icon', function () {
+      var onPopupFormSuccess, url;
```

Comment:
Now I can see that you may simply want to move these vars on 1 *describe* level higher to about duplication of var definition. Also, if you could improve the name of the var `url` it would be great, I would suggest `popupFormUrl`